pkg/network/kuryr.go

```diff
@@ -157,6 +157,15 @@ func validateKuryr(conf *operv1.NetworkSpec) []error {
 		octaviaServiceNet = &octaviaServiceNetObj
 	}
 
+	if kc != nil && kc.EnablePortPoolsPrepopulation && kc.PoolBatchPorts != nil && *kc.PoolBatchPorts > 0 {
```

Comment:
the prepopulation should be independent of the checkings

Reply:
Ah, right. I didn't read the description for it carefully enough.